### PR TITLE
sphinx-gallery fix: don't save plotlyjs in every figure

### PR DESCRIPTION
**Pull request by emmanuelle**

### CODE REVIEW

**Comment by emmanuelle** [comment]
I figured out that it was not a good idea to have every example page weighing 3+ Mo if we want more people to use plotly in docs :-).

File: packages/python/plotly/plotly/io/_base_renderers.py [hunk 1]
```diff
@@ -861,7 +861,7 @@ def render(self, fig_dict):
         filename_html = filename_root + ".html"
         filename_png = filename_root + ".png"
         figure = return_figure_from_figure_or_data(fig_dict, True)
-        _ = write_html(fig_dict, file=filename_html)
+        _ = write_html(fig_dict, file=filename_html, include_plotlyjs=False)
```
Comment (nicolaskruchten) [inline packages/python/plotly/plotly/io/_base_renderers.py:864]: hmm so how does Plotly.js get included?

Comment (nicolaskruchten) [reply packages/python/plotly/plotly/io/_base_renderers.py:864]: the first time I mean

Comment (emmanuelle) [reply packages/python/plotly/plotly/io/_base_renderers.py]: Hum good question. I had checked that with `include_plotlyjs=False` the plotly cdn was included in the html files (which is weird, but it did include the cdn). I switched to `include_plotlyjs='cdn'` instead.

**Comment by nicolaskruchten** [comment]
:dancer:

**Pull request merged**